spec/system/data_migration/sowingseeds_form_submission_spec.rb

```diff
@@ -67,6 +67,7 @@
       click_on "Additional Metadata"
       click_on "Curator Controlled"
       fill_in "ark", with: ark
+      fill_in "publication_year", with: "2023"
```

Comment:
FYI -- This is unrelated to the PR but the test was failing because the fixture is hard coded to 2023 and the system defaults the publication year to the current year to new submissions.